tools/barista/src/main.ts

```diff
@@ -64,9 +82,168 @@ async function buildPages(): Promise<void[]> {
   return Promise.all(files);
 }
 
+function getOverviewSectionItem(
+  filecontent: any,
+  category: string,
+  filepath: string,
+): BaOverviewPageSectionItem {
+  filecontent.properties =
```

Comment:
Please create a new variable for the properties here and don't change the original file content.